### ISSUE TITLE
In apollo-server-azure-functions, getting a typescript error 'Cannot find module './azureFunctions' when compiling.

### ISSUE THREAD

**Issue by andrejpk** [apollographql/apollo-server#2085]
It looks like there's a dependency to an Azure Functions type declaration (src/azureFunctions.d.ts) that's specific to this project but when compiled, that d.ts isn't being put into the dist/ folder so I'm getting an error compiling my project. It still seems to work but I want to get rid of this error.

Here's my demo code: https://github.com/GLRAzure/apollo-graphql-azfunctions-ts-demo 

Maybe I missed something in the installation.. there may be something we can add to tsconfig in the Apollo project to have tsc copy this d.ts into the dist folder.

**Comment by mhabegger** [comment]
It seems, `azureFunctions` is located in the src folder and does not get compiled on build.

Manually editing `azureFunctionApollo.d.ts` to check and importing from `../src/...` make's it work as a temporary workaround.

```
import { HttpContext, FunctionRequest, FunctionResponse } from '../src/azureFunctions';
```

**Comment by andrejpk** [comment]
I played around with it.. if we just rename the azureFunctions.d.ts -> azureFunctions.ts, the compiler will build a d.ts in the output folder. I can make a pull request for this if it would help.

**Comment by jmshal** [comment]
Can be closed now. Fixed in [v2.4.1](https://github.com/apollographql/apollo-server/blob/master/CHANGELOG.md#v241).
